src/HeaderAdapter.vala

```diff
@@ -66,4 +73,13 @@ public class Bluetooth.HeaderAdapter : Gtk.Grid {
             }
         });
     }
+
+    private void update_label () {
+        critical (adapter.discoverable.to_string ());
```

Comment:
Please use `info ()` instead of `critical ()`.

Reply:
Something I forgot to remove after testing. Changed it to info now.